dialogue-clients/metrics.md

```diff
@@ -71,6 +71,11 @@ Dialogue-specific metrics that are not necessarily applicable to other client im
 - `dialogue.client.request.queued.time` tagged `channel-name` (timer): Time spent waiting in the queue before execution.
 - `dialogue.client.request.endpoint.queued.time` tagged `channel-name`, `service-name`, `endpoint` (timer): Time spent waiting in the queue before execution on a specific endpoint due to server QoS.
 - `dialogue.client.request.sticky.queued.time` tagged `channel-name` (timer): Time spent waiting in the sticky queue before execution attempt.
+- `dialogue.client.requests.size` (histogram): Size of requests
```

Comment:
We may want to describe the reporting size threshold in this metric description (though I suppose it doesn't claim to report the size of every request ;-)

Can update the the metric-schema/docs later if needs be.

Reply:
Updated! Yeah, good to call this out... might even be worth renaming the metric to `large_requests` or something to explicitly call out that there's some distinction/threshold that's used for it.

Reply:
I think the current name gives us flexibility if we want to change thresholds later on, I don't mind it as is if you're happy with it. Thanks for updating!